### PR TITLE
fixed component height

### PR DESCRIPTION
**Pull request by jw642459986**
fixed the height of the main component, which used to be 100vh; it should no longer be taking up the whole page

### CODE REVIEW

**Comment by jw642459986** [comment]
@augustdolan Hey August, its ready! though I'm still working on my end points. I'm thinking of doing it without react-router

File: client/src/index.jsx [hunk 1]
```diff
@@ -34,7 +33,8 @@ class App extends React.Component {
     }
 
     componentDidMount() {
-        controllers.getFurnitures()
+        //window.location.pathname.substring(10)
```
Comment (hongmm) [inline client/src/index.jsx:36]: Probably can get rid of or save the info somewhere else if it is not going to be used

File: server/controllers/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
 
 module.exports = {
-    getFurnitures: () => {
-        return axios.get('/api/similarProducts/products/1');
+    getFurnitures: (id) => {
+            return axios.get(`/api/similarProducts/products/${id}`);
     }
 }
```
Comment (hongmm) [inline server/controllers/index.js:7]: Need newline

**Review by hongmm — COMMENTED** [review]
@jw642459986 question: is the code for utilizing url params going to be in this diff? It looks like you made an attempt but left that commented out.

**Comment by jw642459986** [comment]
Yea I'll make a PR for that shortly

File: server/controllers/index.js (hunk 2 above)
Comment (jw642459986) [reply server/controllers/index.js:7]: What do you mean by that?

**Comment by jw642459986** [comment]
I've made new changes to the endpoints

**Pull request closed without merging**